plugin/evm/vm.go

```diff
@@ -565,6 +626,39 @@ func (vm *VM) CreateStaticHandlers() (map[string]*commonEng.HTTPHandler, error)
  ******************************************************************************
  */
 
+// conflicts returns an error if [inputs] conflicts with any of the atomic inputs contained in [ancestor]
+// or any of its ancestor blocks going back to the last accepted block in its ancestry. If [ancestor] is
+// accepted, then nil will be returned immediately.
+// If the ancestry of [ancestor] cannot be fetched, then [errRejectedParent] may be returned.
+func (vm *VM) conflicts(inputs ids.Set, ancestor *Block) error {
```

Comment:
This is not needed (related to atomic UTXOs).

Reply:
done